frontend/lib/ui/Form/Form.tsx

```diff
@@ -109,6 +112,11 @@ export const Form = React.forwardRef<HTMLFormElement, FormProps>(({ children, sk
     submitButton.current = innerRef.current?.querySelector("button[type=submit]") as HTMLButtonElement | null;
   }, [children, skip]);
 
+  // TODO: Need to perform this also when modal closes
```

Comment:
@lkleuver Do you have any suggestion how to focus the form title again after a modal closes (onClose/onCancel)?

Reply:
@Lionqueen94 shouldn't we return the focus to where it was when opening the modal? (no clue whether this will make implementation harder or easier)